components/PostVisitActionInput.tsx

```diff
@@ -84,6 +92,20 @@ export const PostVisitActionInput: React.FunctionComponent<PostVisitActionInputP
           onValueChange(newValue);
         }}
       />
+      {previousPostVisitActions.length > 0 && (
+        <Details
+          summary={{
+            id: "post-visit-actions",
+            value: "Previous post-visit-actions",
+          }}
+        >
+          <List
+            items={previousPostVisitActions.map(
+              (note) => `${note.value}. Created: ${note.createdAt}`
```

Comment:
```suggestion
              (note) => `${note.value} (created on ${note.createdAt})`
```

We don't know what punctuation they'll use, so this feels cleaner.

Reply:
Or even better:

```suggestion
              (note) => `${note.createdAt}: ${note.value}`
```